api/lib/domain/errors.js

```diff
@@ -174,6 +174,12 @@ class SchoolingRegistrationsCouldNotBeSavedError extends DomainError {
   }
 }
 
+class SameNationalStudentIdInOrganizationError extends DomainError {
+  constructor(message = 'Multiplue schooling registrations with same INE') {
```

Comment:
```suggestion
  constructor(message = 'Multiple schooling registrations with same INE') {
```

Reply:
**question** : hmmm est-ce que c'est vraiment ça la description du problème ? parce que quand le contrainte d'unicité [orgaID-userID] est pas respectée (exemple avec les problèmes récents) on a aussi un problème d'unicité mais c'est pas 'Multiple schooling registrations with same INE' non ?

Reply:
J'ai changé pour "INE is already registered in organization". Qu'en penses-tu ?

Reply:
Pour le cas que j'évoque le problème n'est pas ça non plus. Ce serait plutôt "Multiple schooling registrations with same organizationId and userId". Mais du coup ça colle pas avec le cas "Multiple schooling registration with same INE". Je sais pas comment unifier ça.
Du coup je dirais même que `SameNationalStudentIdInOrganizationError` comme nom c'est pas correct. En vrai je mettrais tout dans l'erreur `SchoolingRegistrationCouldNotBeSavedError`.

Reply:
Yes je pense

Reply:
Ok je fais ça 👌